Makefile

```diff
@@ -21,10 +21,18 @@
 #
 ONBLD_TOOLS=/opt/onbld
 
+# Do both debug and non-debug build in uts, in that order, so
+# (a) we know the debug code builds, and (b) install non-debug.
 install: FRC
+	$(ONBLD_TOOLS)/bin/bldenv -d myenv.sh \
```

Comment:
Just a note: it's a no-op for build server https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/openindiana/gfx-drm/Makefile#L78